compose/cli/log_printer.py

```diff
@@ -4,9 +4,12 @@
 import sys
 from itertools import cycle
 
+from six import next
+
 from . import colors
 from .multiplexer import Multiplexer
 from .utils import split_buffer
+from compose import utils
```

Comment:
nope, no reason, I can change it to be more consistent, but we do a bit of both in other modules